libvmi/disk/vbd.c

```diff
@@ -165,6 +165,14 @@ static status_t vbd_qcow2_open(QCowFile *qcowfile, const char *filename)
         /* Check if backing file has absolute path then simply copy it to structure */
         if (!strncmp(backing_file_name, "/", 1)) {
             g_stpcpy(qcowfile->backing_file, backing_file_name);
+        } else if (!strncmp(backing_file_name, "json:", 5)) {
+            char prefix[] = "\"filename\": \"";
+            char *filename = strstr(backing_file_name, prefix);
```

Comment:
Please use a different variable name, there is already a `filename` in this scope as function input.

Reply:
Done